ns/math.n3

```diff
@@ -0,0 +1,239 @@
+# Schema for CWM's mathematical built-ins, SBP 2001-12
+#
+# $Id: math.n3,v 1.12 2010-03-30 15:18:08 timbl Exp $
+
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
+
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix log: <http://www.w3.org/2000/10/swap/log#> .
+@prefix math: <http://www.w3.org/2000/10/swap/math#> .
+@prefix string: <http://www.w3.org/2000/10/swap/string#> .
+@prefix vs: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+
+math:StrictProperty a rdfs:Class;
+    rdfs:label "StrictProperty";
+    rdfs:subClassOf rdf:Property.
+
+math:Value a rdfs:Class; rdfs:label "Value"; 
+   rdfs:subClassOf string:String; 
```

Comment:
```suggestion
   rdfs:subClassOf [owl:unionOf (xsd:decimal xsd:double xsd:float}]; 
```

This may be too complicated for such a vocabulary, but makes it clear that the type is a numeric type and that a value with this domain can be inferred to be a numeric literal.

Reply:
Yes, indeed - it also offers a space for tying in our own type promotion and substitution mechanism